pages/mesosphere/dcos/2.1/release-notes/open-source-attribution/index.md

```diff
@@ -9,184 +9,742 @@ model: /mesosphere/dcos/2.1/data.yml
+|finagle-http-cookie                                                                                                       |18.7.0                                   |<a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|finagle-http2                                                                                                             |18.7.0                                   |<a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|finagle-init                                                                                                              |18.7.0                                   |<a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|finagle-netty3                        
```

Comment:
a whole lot of the errors in this file are due to the usage of plain tags. if we used the markdown equivalent, things would look a tiny bit better.

```
# 🔴 
<a href="https://my.org">Some Content</a>

# ✅ 
[Some Content](https://my.org)
```